### ISSUE TITLE
Hints: Playthrough hints should not hint at multiple Skull Tokens of the same type

### ISSUE THREAD

**Issue by Revvenn** [OoTMM/OoTMM#435]
More or less, if you have Gold Skulls, Swamp Skulls, or Ocean Skulls required and a playthrough hint is going to pull any shuffled skulls there should only be a playthrough hint for *one* of those types each. I've seen some people run seeds where all 4 of their playthrough hints were wasted on Gold Skull Tokens because they had 40 or 50 skulls required and they knew they were required because of the Spiders in the Skull House telling them the reward, of course.

You can know they're required for something more ambiguous on the rewards from just one playthrough hint about skulls imo. You shouldn't have all 4 of your playthrough hints wasted on skulls with skulls shuffled in any capacity.

**Comment by Nax** [comment]
Old issue - do we still want that?

**Comment by Revvenn** [comment]
> Old issue - do we still want that?

Yes. I think it's still valuable. It's a massive waste when you have two playthrough hints (or more) just for Gold, Swamp, or Ocean Skulls.
